support-frontend/assets/pages/[countryGroupId]/checkout.tsx

```diff
@@ -952,13 +962,22 @@ function CheckoutComponent({
 		abParticipations.abandonedBasket === 'variant',
 	);
 
-	const redirectCountryToCountryGroup =
-		abParticipations.redirectCountryToCountryGroup === 'variant';
-
 	return (
 		<CheckoutLayout>
 			<Box cssOverrides={shorterBoxMargin}>
 				<BoxContents>
+					{forcedCountry &&
+						productDescription.deliverableTo?.[forcedCountry] && (
+							<div role="alert" data-qm-error>
```

Comment:
Should this have the QM error data attribute?

Reply:
Nice spot - I can't find any docs on what it does, but it doesn't seem relevant.